docs/concepts/events.md

```diff
@@ -114,20 +110,17 @@ Prefect Cloud provides an interactive dashboard to analyze and take action on ev
 
 ![Event feed](/img/ui/event-feed.png)
 
-## Event feed <span class="badge beta"></span>
+## Event feed
 
 The event feed is the primary place to view, search, and filter events to understand activity across your stack. Each entry displays data on the resource, related resource, and event that took place.
 
 ## Event details
```

Comment:
This header feels unnecessary without an image beneath it.

Reply:
Updated in pushed changes